src/main/kotlin/br/com/creditas/projethom/controller/TeamController.kt

```diff
@@ -0,0 +1,33 @@
+package br.com.creditas.projethom.controller
+
+import br.com.creditas.projethom.dto.TeamForm
+import br.com.creditas.projethom.dto.TeamView
+import br.com.creditas.projethom.service.TeamService
+import org.springframework.transaction.annotation.Transactional
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import javax.validation.Valid
+
+@RestController
+@RequestMapping("/teams")
+class TeamController (
+  private val service: TeamService
```

Comment:
Nada de errado aqui, mas eu particularmente prefiro ser redundante com nomes pra facilitar a leitura, fica a sugestão
```suggestion
  private val teamService: TeamService
```